components/block/element.js

```diff
@@ -9,61 +10,119 @@ export default class Block extends LitElement {
   static styles = [style];
 
   static properties = {
-    blockId: { type: Number},
+    loading: {type: Boolean, state: true},
+    blockId: {type: String},
   };
 
-  renderMenu(){
-    return html`
-      <nav>
-        <button @click=${this.edit}>Edit</button>
-        <button @click=${this.moveUp}>Move Up</button>
-        <button @click=${this.moveDown}>Move Down</button>
-        <button @click=${this.delete}>Delete</button>
-      </nav>
-    `;
+  constructor() {
+    super();
+
+    this.loading = false;
+  }
```

Comment:
You can avoid this entire constructor as `this.loading` is `undefined` by default which when converted to Boolean is same as `false`. If you prefer to explicit it set it you can keep it.

Reply:
The only reason I added the constructor is because I wanted to explicitly make it `false`, rather than have it implicitly converted. I'd prefer to keep it.